configuration.md

```diff
@@ -36,7 +36,9 @@ Below is the server-config.json with all default values
         // Values are "always" | "never" | "if-not-exists"
         create: "if-not-exists"
         // Whether the repository should keep the history
-        "history": false
+        "history": false,
+        // Values can be: "2023.1" | "2024.1"
+        lionWebVersion: "2023.1"
```

Comment:
I think 99% of users will go for the latest version, so I would use 2024.1 in this example

Reply:
Ok, but that makes 2024.1 the default value from now on, as this document describes the default values.